src/generators/index.js

```diff
@@ -10,6 +10,7 @@ module.exports = async (tools) => {
       packageJson: await require('./src/packagejson')(tools, cmd.new),
       config: infra ? await require('./src/infra/config')(tools, cmd.new) : ignore,
       entities: await require('./src/domain/entities')(tools, cmd.new),
+      useCases: await require('./src/domain/usecases/useCases')(tools, cmd.new),
```

Comment:
Why did this stop working? This line was removed in other old versions?

Reply:
I don't know when this line was removed, but without it we won't be able to generate the usecase templates

Reply:
As explained here (https://github.com/herbsjs/herbs-cli/issues/127#issuecomment-1154255711) , this is by design. The generator for use cases (and others based on metadata) only run on `update` command because of its dependencies with the generated project:
When running `herbs new`, you need to accept running `npm install` and `herbs update` to be able to generate these files.
These is the reason why the CLI does not have `@herbsjs\herbs` dependency. Because it doesn't need it. Only the generated project need it.
It would be nice to generate the UCs without needing to run `npm install`, but it proved to be very hard. If someone find a solution, it would be great. But the current PR is not going on this direction.